File: Alley rai/marks.c

#include<stdio.h>
void main () {
	int math,science,english;
	printf("Enter the marks of math :\n");
	scanf("%d",&math);
	printf("Enter the marks of science :\n");
	scanf("%d",&science);
	printf("Enter the marks of english :\n");
	scanf("%d",&english);
	if(math>=40 && science>=40 && english>=40)
	{
		int total=math+science+english;
		int per=total/3;
		if(per>80 && per<=100)
		{
			printf("You have secured A grade");	
		}
	else if(per>=60 && per<80)
	{
		printf("You have secured B grade");
	}else if(per>=40 && per<60)
	{
		printf("You have secured C grade");
	}
	}else{
		printf("You have failed your exam");
	}
}

File: marks.py
english=int(input("Enter the marks of english :"))
science=int(input("Enter the marks of science :"))
math=int(input("Enter the marks of math :"))
if(0<math<100 and 0<science<100 and 0<english<100):
    if(math>=40 and science>=40 and english>=40):
        total=int(english+science+math)
        per=int(total/3)
        if(per>=80 and per<100):
            print("You have secured A grade i.e",per)
        elif(per>=60 and per<80):
            print("You have secured B grade i.e",per)
        elif(per>=40 and per<60):
            print("You have secured C grade i.e",per)
    else:
        print("You have failed your exam.")    
else:
    print("invalid input.")    